drivers/mbox/mbox_nxp_imx_mu.c

```diff
@@ -46,7 +46,7 @@ static int nxp_imx_mu_send(const struct device *dev, uint32_t channel, const str
 	}
 
 	/* Data transfer mode. */
-	if (msg->size != MU_MBOX_SIZE) {
+	if ((msg->size < 0) || (msg->size > MU_MBOX_SIZE)) {
 		/* We can only send this many bytes at a time. */
```

Comment:
It doesn't makes sense to compare msg->size to 0, as it is an unsigned integer underneath.

I think this patch should read just like this 

```
+ 	if (msg->size > MU_MBOX_SIZE) {
```

Reply:
done